controllers/onboarding-controllers.ts

```diff
@@ -0,0 +1,136 @@
+import { Request, Response } from "express"
+import fs from "fs"
+import { v4 as uuidv4 } from 'uuid';
+import { registerSchema, updateSchema } from "../validation-schema/onboarding-schema";
+
+import { I_USER } from "../interfaces/user";
+
+export const RegisterUser = async(req:Request,res:Response) =>{
+    try{
+        
+        const { error } = await registerSchema.validateAsync(req.body);
```

Comment:
Use validation in middlewares

Reply:
Yes sir, I will make all the validations go through middlewares.